alembic/versions/5647b51fe7cb_initial_model_setup.py

```diff
@@ -0,0 +1,93 @@
+    sa.Column('name', sa.String(), nullable=True),
+    sa.Column('drivers', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('driver_ids', postgresql.ARRAY(sa.Integer()), nullable=True),
+    sa.Column('trips', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('times', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('earliest_picks', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('latest_drops', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('miles', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('revenues', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('location_lats', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('location_lons', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('location_labels', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('driver',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('address', sa.String(), nullable=False),
+    sa.Column('capacity', sa.Float(), nullable=True),
+    sa.Column('level_of_service', sa.String(), nullable=False),
+    sa.Column('alternate_early_day', sa.Boolean(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('merge_address',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('address', sa.String(), nullable=False),
+    sa.Column('window', sa.Interval(), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('revenue_rate',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('lower_mileage_bound', sa.Float(), nullable=False),
+    sa.Column('upper_mileage_bound', sa.Float(), nullable=False),
+    sa.Column('level_of_service', sa.String(), nullable=False),
+    sa.Column('base_rate', sa.Float(), nullable=False),
+    sa.Column('revenue_per_mile', sa.Float(), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_table('driver_assignment',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('date', sa.DateTime(), nullable=True),
+    sa.Column('driver_id', sa.Integer(), nullable=False),
+    sa.Column('assignment_id', sa.Integer(), nullable=False),
+    sa.Column('lats', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('lons', postgresql.ARRAY(sa.Float()), nullable=True),
+    sa.Column('trip_ids', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_pu', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_do', postgresql.ARRAY(sa.String()), nullable=True),
+    sa.Column('trip_est_pu', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_sch_pu', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_est_do', postgresql.ARRAY(sa.Interval()), nullable=True),
+    sa.Column('trip_sch_do', postgresql.ARRAY(sa.Interval()), nullable=True),
```

Comment:
expand `pu` `do` to `pickup` `dropoff` and or add `sqlalchemy.Column.comment` properties

Reply:
Addressed in 2b8e7